.github/workflows/release.yml

```diff
@@ -27,18 +29,18 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
           git config --global --add safe.directory /charts
 
-      - name: Copy LICENSE and README.md for packaging
+      - name: Copy LICENSE, EXAMPLES.md and README.md for packaging
         run: |
           cp ./README.md ./traefik/README.md
           cp ./EXAMPLES.md ./traefik/EXAMPLES.md
           cp ./LICENSE ./traefik/LICENSE
 
       - name: Generate default static install
         run: |
-          kustomize build traefik/crds > traefik.yaml
+          kustomize build traefik-crds > traefik.yaml
```

Comment:
Locally, this line exit with this msg:
> Error: must build at directory: not a valid directory: evalsymlink failure on 'traefik-crd' : lstat /home/mloiseleur/git/traefik-helm-chart/traefik-crd: no such file or directory

Reply:
It works. Just don't do any typo while running the command :-): traefik-crd**s**